src/main/java/uk/gov/pay/webhooks/healthcheck/HealthCheckResource.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.pay.webhooks.healthcheck;
+
+import com.google.common.collect.ImmutableMap;
+import io.dropwizard.setup.Environment;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path("/")
+public class HealthCheckResource {
+    private final Environment environment;
+    
+    @Inject
+    public HealthCheckResource(Environment environment) {
+        this.environment = environment;
+    }
+
+    @GET
+    @Path("/")
+    @Produces(APPLICATION_JSON)
+    public Response healthCheck() {
+        var ok = ImmutableMap.of("healthy", "true");
```

Comment:
We can use `Map.of("healthy", "true")` here.

Reply:
done